src/brogue/RogueMain.c

```diff
@@ -579,7 +579,7 @@ void startLevel(short oldLevelNumber, short stairDirection) {
 
     for (i=0; i<DCOLS; i++) {
         for (j=0; j<DROWS; j++) {
-            if (pmap[i][j].flags & VISIBLE) {
+            if (pmap[i][j].flags & ANY_KIND_OF_VISIBLE) {
```

Comment:
Telepathic/clairvoyant visible tiles that have been added to but not yet removed from FOV don't get their flags stored to memory, even though their appearance gets remembered. Then they're treated as blocking when allies are restored, so they're unusable for pathing.

Obviously won't happen often, but I noticed this because of the clairvoyance ring in wizard mode.